doc/catalyst-cli/catalyst-cli.rst

```diff
@@ -0,0 +1,215 @@
+Catalyst CLI
```

Comment:
Maybe the section title could be a bit more descriptive? For a new user an unexplained abbreviation (despite "CLI" being quite established in software circles) might not be the best section title.

cc @josh146

Reply:
Or maybe just "Catalyst Command Line Interface"?

Reply:
I agree, I think it's best to avoid acronyms as much as possible. "Catalyst Command Line Interface" as a section heading would be better, then we can contract it to CLI in the text below.

Reply:
Changes in 5d9d536